build.gradle

```diff
@@ -69,10 +69,10 @@ configure(allprojects) { project ->
   group = 'io.projectreactor.kafka'
 
   repositories {
-    maven { url 'https://repo.spring.io/libs-release' }
-    maven { url 'https://repo.spring.io/libs-snapshot' }
     mavenCentral()
     jcenter()
+    maven { url 'https://repo.spring.io/libs-release' }
+    maven { url 'https://repo.spring.io/libs-snapshot' }
   }
```

Comment:
you should switch also to `https://repo.spring.io/release` and `https://repo.spring.io/snapshot`

Reply:
I kept it consistent with reactor-core:
https://github.com/reactor/reactor-core/blob/669df3dbe7b7907288fe248bf1ee84a3dfcaf3fc/build.gradle#L131

Reply:
reactor-core is actually wrong, it is preferable to use the ones without `libs-`

Reply:
> January 6, 2021
> 
> We will no longer support anonymous download of 3rd-party artifacts from repo.spring.io. The /libs-snapshot, /libs-milestone, and /libs-release repositories, which the Spring Team uses to build and release Spring projects, will require authentication.
The `/snapshot`, `/milestone`, and /release repositories will remain available, but please fetch our releases from a central repository.

source: https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020